imagecraft/image_definition.py

```diff
@@ -177,6 +178,7 @@ def __init__(  # noqa: PLR0913
                     names=seed_names,
                     pocket=seed_pocket,
                 ),
+                sources_list_deb822=True,  # Always set it to true for now
```

Comment:
Yes, this is "hack" for now. The "base" thing is a really tricky thing because the craft framework is expecting this base to exists and to be usable to build. How do we build a newer series on an older one in this case? 

But otherwise I agree we could probably look at some value in the provided YAML and decide for the user if this should be set True/False.

Reply:
Or we could the `version` field?